google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1379,7 +1379,7 @@ def compose sources, destination, acl: nil, encryption_key: nil
             updater.check_for_changed_metadata!
           end
           gapi = service.compose_file name, sources, destination,
-                                      destination_gapi, options
+                                      destination_gapi, **options
```

Comment:
I'd prefer to pass keyword arguments directly here, since there is no `delete_if` on the options hash:

```rb
          acl_rule = File::Acl.predefined_rule_for acl
          gapi = service.compose_file name, sources, destination, destination_gapi, acl: acl_rule,
                                                                                    key: encryption_key,
                                                                                    user_project: user_project
```